### DISCUSSION TITLE
How do I disable sourcemap upload/release?

### DISCUSSION THREAD

**Discussion by wawhal** [getsentry/support#42] [Support]
I have disabled the sentry sourcemap release by removing the webpack plugin as per the [docs](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/?_ga=2.4445171.1599458041.1658306950-1412059903.1658306950#disable-sentrywebpackplugin). When I build my nextJS application, the release upload is attempted anyway and I get this error:

```
Sentry CLI Plugin: Command failed: /home/rishichandra/go/src/github.com/hasura/temp-hasura-changelog/ui/node_modules/@sentry/cli/sentry-cli releases new tCEEhMylNCa2YX_3cNQls
error: API request failed
  caused by: sentry reported an error: Authentication credentials were not provided. (http status: 401)

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.

Sentry CLI Plugin: Command failed: /home/rishichandra/go/src/github.com/hasura/temp-hasura-changelog/ui/node_modules/@sentry/cli/sentry-cli releases new tBEEhWylNCa2YX_3cNQls
error: API request failed
  caused by: sentry reported an error: Authentication credentials were not provided. (http status: 401)

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.


> Build failed because of webpack errors
```

I don't want my sourcemap to be uploaded and only want to use Sentry for error alerts. How can I achieve this?
Found a related thread in the archived forums: https://forum.sentry.io/t/how-could-i-disable-auto-release-for-sentry-nextjs/13923

**Comment by Kobby-Bawuah** [comment]
Hello @wawhal, 

Could you confirm [this](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#disable-sentrywebpackplugin) did not work? Also, could you confirm you tried passing `dryRun` as a webpack plugin option?
